internal/propiedadesTemporada.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+type PropiedadesTemporada struct {
```

Comment:
Tratad de reflejar en los nombres de las clases lo que estáis modelando. En este caso, ¿se trata de las propiedades de un equipo en una temporada?

Reply:
Quizás no he indicado un nombre muy correcto a la estructura. Esta se basa en aquellas propiedades de un jugador en una temporada que, tal y como se ha mencionado más arriba, son las partes mutables de este.

Reply:
Trata de cambiar el nombre para que diga eso.

Reply:
Acabo de cambiar los nombres para que sean más acordes al modelo del dominio, le comento ahora en un comentario más general!